src/XAMLTools.Core/XAMLCombine/XAMLCombiner.cs

```diff
@@ -300,12 +369,26 @@ private static void ChangeNamespacePrefix(XmlElement element, string oldPrefix,
                         attr.Prefix = newPrefix;
                     }
 
-                    // Check {x:Type {x:Static in attributes values
-                    // TODO: Is any other???
-                    if ((attr.Value.Contains("{x:Type") || attr.Value.Contains("{x:Static"))
-                        && attr.Value.Contains(oldStringSpaced))
+                    if (attr.Value.Contains(oldStringSpaced))
```

Comment:
We should check attr.Value first to make sure it's not null or whitespace?

Reply:
Cannot be null here by design, i believe